_posts/2024-06-20-pyos-guide-to-scipy.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: single
```

Comment:
Just a note - the single layout is an html page designed content blog type content. it should support a TOC. the splash page is designed for a landing page - a more flowing / scrolling page that doesn't necessarily have indexable sections.

Reply:
that makes sense, thanks!